modules/dxp/apps/portal-workflow/portal-workflow-kaleo-forms-test/src/testFunctional/paths/ProcessBuilderKaleoDesignerReact.path

```diff
@@ -166,7 +166,16 @@
 	<td>//input[@type='checkbox' and @value ='${key_checkboxValue}']</td>
 	<td></td>
 </tr>
-
+<tr>
+	<td>BUTTON_SOURCE_DIAGRAM</td>
+	<td>//*[not(self::div)][contains(@class,'btn-secondary')][normalize-space()=''][not(contains(@class,'navbar'))]</td>
+	<td></td>
+</tr>
+<tr>
+	<td>VIEW_TEXT_SOURCE_DIAGRAM_BUTTON</td>
```

Comment:
Lets see if this is really needed, check Tooltip.path file. Thanks

Reply:
Done